app/src/components/Base.tsx

```diff
@@ -44,33 +56,74 @@ export const Base = () => {
   }, [pathWithoutLocale]);
 
   return (
-    <Box minHeight="100vh" flex={1} display="flex" flexDirection="column">
-      <Box display="flex" flex={1}>
-        <Outlet />
+    ( isSmall
+      ?
+      <Box minHeight="90vh" flex={1} display="flex" flexDirection="column"> 
+        <Paper
+          sx={{ position: "sticky", top: 0, zIndex: 1100 }}
+          elevation={2}
+        >
+          <IconButton onClick={() => setDrawerState(true)}>
+            <Menu sx={{ fontSize: 40, color: "black" }} />
+          </IconButton>
+        </Paper>
+        <Drawer
+          open={drawerState}
+          anchor="left"
+          onClose={() => setDrawerState(false)}
+        >
+          <List>
+            {ROUTES.map((r) => {
+              const Icon = ICONS[r];
+              const route = ROUTE_NAME_TO_ROUTE[r];
```

Comment:
It's a bit confusing that `r` comes from `ROUTES` but apparently needs to be transformed into `route`.
I think `r` being `route`, `ROUTE_NAME_TO_ROUTE` being `ROUTE_TO_PATH`, and `route`/`routeWithLocale` being `path`/`pathWithLocale` might be better? Or something like that. Not sure if that's consistent with the rest of the codebase though.
I understand that this is just coming from the existing code.

Reply:
I've changed `r` to routeName to be a bit more descriptive. However, it appears that having `${locale}/${route}` is fairly consistently used among the rest of the codebase.